common/src/main/java/com/wynntils/screens/maps/widgets/PoiManagerWidget.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.maps.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.features.map.MapFeature;
+import com.wynntils.models.map.pois.CustomPoi;
+import com.wynntils.screens.maps.PoiCreationScreen;
+import com.wynntils.screens.maps.PoiManagementScreen;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.mc.RenderedStringUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import java.util.List;
+import java.util.Optional;
+import net.minecraft.client.gui.components.AbstractWidget;
+import net.minecraft.client.gui.components.Button;
+import net.minecraft.client.gui.narration.NarrationElementOutput;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.network.chat.Component;
+
+public class PoiManagerWidget extends AbstractWidget {
+    private CustomPoi poi;
+    private Button editButton;
+    private Button deleteButton;
+    private Button upButton;
+    private Button downButton;
+    private int row;
+    private CustomColor color;
+    private int spacingMultiplier = 20;
+    private PoiManagementScreen managementScreen;
+    private List<CustomPoi> pois;
+
+    public PoiManagerWidget(
+            float x, float y, int width, int height, CustomPoi poi, int row, PoiManagementScreen managementScreen) {
+        super((int) x, (int) y, width, height, Component.literal(poi.getName()));
+        this.poi = poi;
+        this.row = row;
+        this.managementScreen = managementScreen;
+
+        pois = Managers.Feature.getFeatureInstance(MapFeature.class).customPois.get();
+
+        color = CustomColor.fromInt(0xFFFFFF);
```

Comment:
We have CommonColors.WHITE for this purpose

Reply:
Here, and below.. resolved but not changed?